Makefile

```diff
@@ -112,7 +112,19 @@ checktidy:
 checkgenerate:
 	./etc/checkgenerate.sh
 
-generate: $(SUBDIRS)
+generate: gogo $(SUBDIRS)
+	$(GO_GENERATE) \
+		./v1/services/meta \
+		./v1/services/storage \
+		./tsdb \
+		./chronograf/bolt/internal \
+		./storage/reads/datatypes \
+		./influxql/query \
+		./pkg/tracing/wire
```

Comment:
A blanket `go generate ./...` hits errors because of something in `chronograf/dist`... instead of enumerating these here & having to remember to update / add to them, is there a way to say "all modules except for `X`" when passing to `go generate`?

Reply:
Nevermind, found it! Edit incoming...